src/test/java/plugins/FolderPluginTest.java

```diff
@@ -24,24 +24,31 @@
 
 package plugins;
 
+import org.apache.commons.io.IOUtils;
 import org.jenkinsci.test.acceptance.Matchers;
 import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
 import org.jenkinsci.test.acceptance.junit.WithPlugins;
 import org.jenkinsci.test.acceptance.plugins.credentials.CredentialsPage;
 import org.jenkinsci.test.acceptance.plugins.credentials.ManagedCredentials;
 import org.jenkinsci.test.acceptance.plugins.credentials.UserPwdCredential;
 import org.jenkinsci.test.acceptance.po.*;
+import org.junit.Assert;
```

Comment:
```suggestion
```
The pre-existing tests already use org.hamcrest.MatcherAssert#assertThat(), let's stick to one #assertThat.